<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #9575CD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_Agregar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 227);
text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(255, 255, 255);
border-radius:65px;
border:10px solid grey;
</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_calificacion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MV Boli&quot;;
background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
border-radius:25px;
border:10px solid grey;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>Calificacion:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_promedio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MV Boli&quot;;
background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
border-radius:25px;
border:10px solid grey;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>Promedio:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Guardar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 227);
text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(255, 255, 255);
border-radius:65px;
border:10px solid grey;
</string>
    </property>
    <property name="text">
     <string>GUARDAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cargar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(176, 176, 227);
text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(255, 255, 255);
border-radius:65px;
border:10px solid grey;
</string>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string>Calificaciones:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_listacalificaciones">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MV Boli&quot;;
background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
border-radius:25px;
border:10px solid grey;
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
